server/public/layout.css

```diff
@@ -101,3 +108,152 @@ body {
+  height: var(--rsbs-overlay-h);
+  transform: translate3d(0, var(--rsbs-overlay-translate-y), 0);
+  will-change: height;
+}
+
+[data-rsbs-overlay],
+[data-rsbs-root]::after {
+  max-width: var(--rsbs-max-w);
+  margin-left: var(--rsbs-ml);
+  margin-right: var(--rsbs-mr);
+}
+
+[data-rsbs-overlay],
+[data-rsbs-backdrop],
+[data-rsbs-root]::after {
+  z-index: 3;
+  overscroll-behavior: none;
+  touch-action: none;
+  position: fixed;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  user-select: none;
+  -webkit-tap-highlight-color: transparent;
+  -webkit-touch-callout: none;
+}
+
+[data-rsbs-overlay]:focus {
+  outline: none;
+}
+
+[data-rsbs-is-blocking="false"] [data-rsbs-overlay] {
+  box-shadow:
+    0 -5px 60px 0 rgba(38, 89, 115, 0.11),
+    0 -1px 0 rgba(38, 89, 115, 0.05);
+}
+
+[data-rsbs-backdrop] {
+  top: -60px;
+  bottom: -60px;
+  background-color: var(--rsbs-backdrop-bg);
+  will-change: opacity;
+  cursor: pointer;
+  opacity: 1;
+}
+
+[data-rsbs-is-dismissable="false"] [data-rsbs-backdrop] {
+  cursor: ns-resize;
+}
+
+[data-rsbs-root]::after {
+  content: "";
+  pointer-events: none;
+  background: var(--rsbs-bg);
+  height: 1px;
+  transform-origin: bottom;
+  transform: scale3d(1, var(--rsbs-antigap-scale-y), 1);
+  will-change: transform;
+}
+
+[data-rsbs-footer],
+[data-rsbs-header] {
+  flex-shrink: 0;
+  cursor: ns-resize;
+  padding: 16px;
+}
+
+[data-rsbs-header] {
+  text-align: center;
+  user-select: none;
+  box-shadow:
+    0 1px 0
+    rgba(46, 59, 66, calc(var(--rsbs-content-opacity) * 0.125));
+  z-index: 1;
+  padding-top: calc(20px + env(safe-area-inset-top));
+  padding-bottom: 8px;
+}
+
+[data-rsbs-header]::before {
+  position: absolute;
+  content: "";
+  display: block;
+  width: 36px;
+  height: 4px;
+  top: calc(8px + env(safe-area-inset-top));
+  left: 50%;
+  transform: translateX(-50%);
+  border-radius: 2px;
+  background-color: var(--rsbs-handle-bg);
+}
+
+@media (min-resolution: 2dppx) {
+  [data-rsbs-header]::before {
+    transform: translateX(-50%) scaleY(0.75);
+  }
+}
+
+[data-rsbs-has-header="false"] [data-rsbs-header] {
+  box-shadow: none;
+  padding-top: calc(12px + env(safe-area-inset-top));
+}
+
+[data-rsbs-scroll] {
+  flex-shrink: 1;
+  flex-grow: 1;
+  -webkit-tap-highlight-color: revert;
+  -webkit-touch-callout: revert;
+  user-select: auto;
+  overflow: auto;
+  overscroll-behavior: contain;
+  -webkit-overflow-scrolling: touch;
+}
+
+[data-rsbs-scroll]:focus {
+  outline: none;
+}
+
+[data-rsbs-content] {
+  overflow: hidden;
+}
+
+[data-rsbs-has-footer="false"] [data-rsbs-content] {
+  padding-bottom: env(safe-area-inset-bottom);
+}
+
+[data-rsbs-footer] {
+  box-shadow:
+    0 -1px 0 rgba(46, 59, 66, calc(var(--rsbs-content-opacity) * 0.125)),
+    0 2px 0 var(--rsbs-bg);
+  overflow: hidden;
+  z-index: 1;
+  padding-bottom: calc(16px + env(safe-area-inset-bottom));
+}
+
+[data-rsbs-state="closed"],
+[data-rsbs-state="closing"] {
+  pointer-events: none;
+}
```

Comment:
are these all styles for a library? i'm surprised the library didn't come with its own styles. If they supply a separate stylesheet it would be good to link that in your HTML from a CDN instead. 

If you need to override some of the styles, I recommend putting that in an `overrides.css` files since a lot of these styles aren't related to layout and it would be easier for another developer to know that some of the default styles of the library are being changed.

In any case, these should probably be in a separate stylesheet since they're all for a specific part of the app